main.tf

```diff
@@ -93,14 +93,6 @@ resource "confluent_service_account" "service_accounts" {
   description  = "${each.value} service account"
 }
 
-# Service Account Role Bindings
-resource "confluent_role_binding" "service_accounts_cluster_role_binding" {
-  for_each    = toset(local.service_accounts)
-  principal   = "User:${confluent_service_account.service_accounts[each.value].id}"
-  role_name   = "CloudClusterAdmin"
-  crn_pattern = confluent_kafka_cluster.cluster.rbac_crn
-}
-
```

Comment:
As local.service_accounts is a union of readers and writers we don't need to grant ClusterAdmin role to them.